app/src/main/java/cm/aptoide/pt/app/view/GridAppWidget.java

```diff
@@ -51,21 +46,18 @@ public GridAppWidget(View itemView) {
     ImageLoader.with(context)
         .load(pojo.getIcon(), icon);
 
-    int downloads = displayable.isTotalDownloads() ? pojo.getStats()
-        .getPdownloads() : pojo.getStats()
-        .getDownloads();
-
     name.setText(pojo.getName());
-    this.downloads.setText(
-        AptoideUtils.StringU.withSuffix(downloads) + context.getString(R.string._downloads));
-    ratingBar.setRating(pojo.getStats()
-        .getRating()
-        .getAvg());
-    tvStoreName.setText(pojo.getStore()
-        .getName());
-    tvAddedTime.setText(AptoideUtils.DateTimeU.getInstance(getContext())
-        .getTimeDiffString(context, pojo.getAdded()
-            .getTime(), getContext().getResources()));
+
+    try {
+      DecimalFormat oneDecimalFormatter = new DecimalFormat("#.#");
+      rating.setText(oneDecimalFormatter.format(pojo.getStats()
+          .getRating()
+          .getAvg()));
+    } catch (Exception e) {
+      rating.setText(R.string.appcardview_title_no_starts);
```

Comment:
Shouldn't the string be called ...no_stars?

Reply:
done